doc/docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,37 @@
+
+= Neo4j Spark Connector v{docs-version}
+:toc:
+:toclevels: 5
+:sectids:
+:sectlinks:
+:sectnums:
+:sectnumlevels: 5
+:env-docs: true
```

Comment:
I think it should be defined in the playbook otherwise the conditional blocks are not very useful because the value will always be true